resources/identity_validation/models/multiple_components.yml

```diff
@@ -0,0 +1,49 @@
+allOf:
```

Comment:
Can this file be renamed? It will cause confusion with https://github.com/lob/lob-openapi/blob/main/resources/us_verifications/models/multiple_components.yml in both this repo and the SDK repos.